react/components/Promotions/ArchivedPromotionsList.js

```diff
@@ -178,6 +204,11 @@ const ArchivedPromotionsList = ({
   const { navigate } = useRuntime()
   const [unarchivePromotion] = useMutation(unarchivePromotionById)
 
+  let tableSchema = getTableSchema(intl)
+
+  if (archivedPromotions.every(p => p.effectType))
```

Comment:
Can you add a `{}` after `if` statement?
It's a detail and work in both ways, but it's error prone. Someone can into code and just add a new line after `if` and accidentally forget `{}` adding bugs.

```suggestion
  if (archivedPromotions.every(p => p.effectType)) {
      delete tableSchema.properties.legacy
  }
```